### PR TITLE
Bug fix: random seed for keras

### LINKED ISSUE #388: Episode 2 random seed
In episode 2 of the workshop, in step 4 of the workflow two random seeds are set: 

```
from numpy.random import seed
seed(1)
from tensorflow.random import set_seed
set_seed(2)
```

However, the tensorflow.random.set_seed is not the correct one to set, right now the results are not consistent when rerunning the entire workflow

### PR DESCRIPTION
**Pull request by laurasootes**
This PR fixes a bug in step 4 of episode 2. Here two seeds are set to ensure consistent results. However the second seed is not correct or depricated
`from tensorflow.random import set_seed
set_seed(2)`

instead the `keras.utils.set_random_seed` should be used to ensure consistent results. The first seed from numpy is also still neccesary.

fixes #388

### CODE REVIEW

**Review by svenvanderburg — APPROVED** [review]
Thanks @laurasootes for your great contribution!
